bundle-workflow/src/manifests/typechecked.py

```diff
@@ -0,0 +1,111 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Not sure but can we have the name of the file as `typechecker` as `typechecked` is an internal keyword?

Reply:
Renamed to match the class name, `dataclass_typechecked`.